cmd/notation/sign.go

```diff
@@ -94,13 +86,9 @@ Example - [Experimental] Sign an OCI artifact and use OCI artifact manifest to s
 			if opts.ociLayout {
 				opts.inputType = inputTypeOCILayout
 			}
-			return experimental.CheckFlagsAndWarn(cmd, "signature-manifest", "oci-layout")
+			return experimental.CheckFlagsAndWarn(cmd, "allow-referrers-api", "oci-layout")
```

Comment:
Don't think we need API suffix in the flag. `--allow-referrers` would be my preference.

Reply:
> Don't think we need API suffix in the flag. `--allow-referrers` would be my preference.

@sajayantony Just an open discussion, since the `referrers tag schema` also has the word `referrers` in its name, probably we should keep the `api` suffix in the flag? @shizhMSFT @yizha1 @FeynmanZhou Do you have any preferences on this one?

Reply:
As per call. This is to disambiguate between referrers tag schema vs API and hence having the api suffix makes this clearer. 
With that in mind, I agree that `--allow-referrers-api` LGTM.